main/resources/com/google/refine/grel/FunctionDescription.properties

```diff
@@ -120,7 +120,7 @@ fun_jsonize=Quotes a value as a JSON literal value.
 fun_length=Returns the length of string s as a number, or the size of array a, meaning the number of objects inside it. Arrays can be empty, in which case length() will return 0.
 fun_slice=Given a string, returns the substring starting from character index from, and up to character index to. If the to argument is omitted, will output to the end of s. Remember character indices start from zero. Given an array, returns a sub-array from the first index provided up to and including the last index provided. If the to value is omitted, it is understood to be the end of the array. Slice only.
 fun_time_since_unix_epoch_to_date=Returns a number converted to a date based on Unix Epoch Time. The number can be Unix Epoch Time in one of the following supported units: second, millisecond, microsecond. Defaults to ''second''.
-fun_to_date=Returns the inputted object converted to a date object. Without arguments, it returns the ISO 8601 extended format. With arguments, you can control the output format. With monthFirst: set false if the date is formatted with the day before the month. With formatN: attempt to parse the date using an ordered list of possible formats. Supply formats based on the SimpleDateFormat syntax: <a href="http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html">SimpleDateFormat</a>.
+fun_to_date=Returns the date object from input string.The arguments available to the toDate function control how the inputted object is interpreted as a date.The function can also parse dates in non-default locale like we can give in the first string can be a locale identifier instead of a format string and so when a locale identifier is provided, the parsing function will use the formatting rules for that locale, rather than the default formatting rules and it will give us the output date object.
```

Comment:
toDate can take a number as an input as well so the first sentence
>Returns the date object from input string

should reflect this (e.g. "input string or number")

There should be a space after each full stop

I think the third sentence is quite long - I'd suggest either trying to re-phrase more concisely or splitting into two sentences


Also suggest we avoid use of a 

I'd suggest it could also break into two sentences so for example:
"also parse dates in non-default locale. For example we can give in the"